backend/openapi/openapi.json

```diff
@@ -1248,36 +1245,38 @@
             "allOf": [{ "$ref": "#/components/schemas/ProjectStatus" }],
             "default": "LOADING"
           },
-          "taxonomy_name": { "type": "string", "title": "Taxonomy Name" },
-          "branch_name": { "type": "string", "title": "Branch Name" },
+          "taxonomyName": { "type": "string", "title": "Taxonomyname" },
+          "branchName": { "type": "string", "title": "Branchname" },
           "description": { "type": "string", "title": "Description" },
-          "is_from_github": { "type": "boolean", "title": "Is From Github" },
-          "created_at": {
+          "ownerName": { "type": "string", "title": "Ownername" },
+          "isFromGithub": { "type": "boolean", "title": "Isfromgithub" },
+          "createdAt": {
             "type": "string",
             "format": "date-time",
-            "title": "Created At"
+            "title": "Createdat"
           },
-          "github_checkout_commit_sha": {
+          "githubCheckoutCommitSha": {
             "anyOf": [{ "type": "string" }, { "type": "null" }],
-            "title": "Github Checkout Commit Sha"
+            "title": "Githubcheckoutcommitsha"
```

Comment:
a bit weird, but not really important

Reply:
Tracked in: https://github.com/openfoodfacts/taxonomy-editor/issues/437